packages/neo-one-website/src/utils/getBlogs.ts

```diff
@@ -75,11 +77,14 @@ const getBlog = async (blogFile: string): Promise<BlogInfo> => {
   const contents = await fs.readFile(path.resolve(BLOG_SOURCE, blogFile), 'utf8');
   const blog = matter(contents);
   const blogHeader = blog.data as MDBlogHeader;
+  const blogSourceLessDirname = BLOG_SOURCE.replace(path.resolve(__dirname, '..', '..', '..', '..'), '');
```

Comment:
Use `path.relative` to get a relative path. Something like `path.relative(path.resolve(__dirname, '..', '..', '..', '..'), BLOG_SOURCE)`